src/asmcnc/comms/user_settings_manager.py

```diff
@@ -47,9 +47,8 @@ class UserSettingsManager(EventDispatcher):
             'value': True
         },
         'interrupt_bars_active': {
-            'description': 'When activated, hitting the interrupt bars will trigger an alarm! '
-                           'Only deactivate this if your interrupt bar switches are broken!',
-            'title': 'Interrupt bars activated',
+            'description': 'When activated, the interrupt bars will pause the machine when depressed.',
```

Comment:
How come the description here changed?

Reply:
I think the previous description was put in place temporarily by Marc, the one I put is from Aaron